src/libraries/System.Private.CoreLib/src/System/Globalization/CultureInfo.Unix.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+
+namespace System.Globalization
+{
+    public partial class CultureInfo : IFormatProvider
+    {
+        internal static CultureInfo GetUserDefaultCulture()
+        {
+            if (GlobalizationMode.Invariant)
+                return CultureInfo.InvariantCulture;
+
+            CultureInfo cultureInfo;
+            string? localeName;
+            if (CultureData.GetDefaultLocaleName(out localeName))
+            {
+                Debug.Assert(localeName != null);
+                cultureInfo = GetCultureByName(localeName);
+            }
+            else
+            {
+                cultureInfo = CultureInfo.InvariantCulture;
+            }
+
+            return cultureInfo;
+        }
+
+        private static CultureInfo GetUserDefaultUICulture()
+        {
+            Debug.Assert(!GlobalizationMode.UseNls);
```

Comment:
This one too?

Reply:
I believe I just removed it as well, but I'll double check. Thanks.